modules/dynamic-operator/config/manager/kustomization.yaml

```diff
@@ -22,10 +22,10 @@ secretGenerator:
 images:
 - name: apiserver
   newName: quay.io/kuberlogic/dynamic-apiserver
-  newTag: 0.0.11
+  newTag: 0.0.12
```

Comment:
Why is it changed?

Reply:
It's not me, something updating `kustomization.yaml`, I can fix it by hands, but chances are high what everyone will hit same issue :)